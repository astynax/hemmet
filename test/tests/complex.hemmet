form:search-form$theme>input.query^red-text>(div.help~hidden_t)+span.submit&button~text_small>.hint
