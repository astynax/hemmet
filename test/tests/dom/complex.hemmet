#container>.nav>(ul.menu>li.item>(img.preview/)+li.item)+#content.width-800.selected
