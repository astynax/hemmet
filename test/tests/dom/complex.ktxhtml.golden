div {
    id = "container"
    div {
        classes = setOf("nav")
        ul {
            classes = setOf("menu")
            li {
                classes = setOf("item")
                img {
                    classes = setOf("preview")
                }
            }
            li {
                classes = setOf("item")
            }
        }
    }
    div {
        id = "content"
        classes = setOf("width-800", "selected")
    }
}
